airflow/www/views.py

```diff
@@ -528,8 +528,8 @@ def code(self, session=None):
                 code = f.read()
             html_code = highlight(
                 code, lexers.PythonLexer(), HtmlFormatter(linenos=True))
-        except OSError as e:
-            html_code = str(e)
+        except (OSError, AttributeError) as e:
```

Comment:
I had to add this for test to pass.
io.open(f, mode=mode) line in www/utils.py l 222 fails with AttributeError
    with wwwutils.open_maybe_zipped(dag.fileloc, 'r') as f:
AttributeError: __enter__